### ISSUE TITLE
DS Examples should have an `h1` page heading when shown in their own tab (DAC usability)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2254]
When you select 'Open this example in a new tab', the contents of the new tab should have a h1 level page heading to help user orientation.

(Page 89-90 of June 2022 DAC DS report...)

---

## Section Missing Heading (Usability)

Sections of the page do not have unique headings to separate them from other content.

### Reference:
Usability & GOV.UK Design System requirement

### Issue ID: DAC_Usability_08
URL: https://ons-design-system.netlify.app/components/accordion/examples/accordion/
Page title: Example
Journey: Journey 1, Step 1, b

Some example pages do not have a main heading that introduce the page content to users. Main headings help screen reader users understand the purpose of the page, and quickly orientate themselves where they are within the service.

### Screen reader comments: 
“When testing the headings on the ‘example’ page I found that there are no headings at level 1. As well as contravening the GOV.UK Design System rules regarding page title and headings at level 1 matching, pages without H1’s are problematic as they indicate main content on the page.”

### Solution:
We advise adding a main heading to the service page. Main headings help understand the content within the page and are also used for navigating the page.
